src/khiva/matrixInternal.h

```diff
@@ -45,6 +112,40 @@ bool tileIsFarFromDiagonal(long bandSize, long numRows, long row, long numColumn
  */
 af::array generateMask(long m, long numRows, long row, long numColumns, long column, long nTimeSeries = 1);
 
+/**
+ * @brief Calculates the Mueen distance.
```

Comment:
Change this to calculate the Mueen similarity search. See #107. When this PR gets merged, please close the mentioned issue.